docs/release-notes/highlights-2.0.0.asciidoc

```diff
@@ -23,3 +23,9 @@ Starting with ECK 2.0 the operator can make Kubernetes Node labels available as
 ==== Smoother cluster operations using node lifecycle APIs
 
 When orchestrating Elasticsearch version 7.15.2 or later ECK will use the new link:https://www.elastic.co/guide/en/elasticsearch/reference/current/node-lifecycle-api.html[node lifecycle APIs] to orchestrate rolling upgrades and scale downs to make Elasticsearch aware of impending temporary or permanent shutdown of nodes.
+
+[float]
+[id="{p}-200-known-issues"]
+=== Known issues
```

Comment:
```suggestion
=== Known issue
```

Reply:
I think I keep `known issues` because it such an established term. Hope that's OK.